sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/TrampolineUtil.scala

```diff
@@ -172,4 +173,10 @@ object TrampolineUtil {
   def getSparkConf(spark: SparkSession): SQLConf = {
     spark.sqlContext.conf
   }
+
+  def toLogicalPlan(df: DataFrame): LogicalPlan = df.logicalPlan
```

Comment:
nit: These are only needed for testing. Would it be better to have a TestTrampolineUtil in the test jar that would expose these instead?